crates/rune/src/options.rs

```diff
@@ -14,6 +17,19 @@ pub struct Options {
     pub bytecode: bool,
 }
 
+impl FromStr for Options {
+    type Err = ConfigurationError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let mut opt = Options::default();
+        let it = s.split(',');
+        for o in it {
+            opt.parse_option(o)?;
+        }
+        Ok(opt)
```

Comment:
@udoprog this the code related to parsing the options separated by `,` so there is no need to make it as `Vec`.